sound/sequences/00_sound_player.s

```diff
@@ -6986,6 +6986,42 @@ sound_ref .sound_menu_power_meter
 sound_ref .sound_menu_camera_buzz
 sound_ref .sound_menu_camera_turn
 
+// Crash screen error sfx
+
+sound_ref .sound_menu_critical_error
```

Comment:
This should not have any whitespace above it; I can see this very easily causing problems for people. (Just add that comment to the same `sound_ref` line.)

Reply:
Will do.